### PR TITLE
Update jwt-token.adoc

### PR DESCRIPTION
**Pull request by quanchentg**

### CODE REVIEW

File: modules/user-access/pages/jwt-token.adoc [hunk 1]
```diff
@@ -492,3 +492,63 @@ gadmin restart -y
 ====
 Existing environment variable may have some influence on the current use of TigerGraph.
 ====
+
+== Usage of GSQL JWT Token
+Starting from 4.1.0, the token generated by GSQL API is JWT Token and can be used on GSQL and RESTPP server for authentication.
```
Comment (arun-TG-PM) [inline modules/user-access/pages/jwt-token.adoc]: Lets add as a special note : Start with "As of version 4.1.0"

File: modules/user-access/pages/jwt-token.adoc [hunk 2]
```diff
@@ -492,3 +492,63 @@ gadmin restart -y
...
+
+=== create JWT Token
+
+[console]
+----
+POST :14240/gsql/v1/tokens payload '{"secret":"<secret>", "graph":"<graph name>, lifetime: <number in ms>"}'
+----
+
+ * `secret`: the `secret` represents the user who owns so there is no need user login if secret set in the payload
```
Comment (arun-TG-PM) [inline modules/user-access/pages/jwt-token.adoc]: For more clarity - The secret represents the user. If a secret is provided in the payload, user login is not required. If no secret is specified in the payload, a secret will be automatically generated.

File: modules/user-access/pages/jwt-token.adoc [hunk 3]
```diff
@@ -492,3 +492,63 @@ gadmin restart -y
...
+    if there is no secret specified in the payload a secret will be generated automatically
+                  
+ * `graph`: the `graph` means the scope the token can be used
```
Comment (arun-TG-PM) [inline modules/user-access/pages/jwt-token.adoc]: For more clarity. Try to split into simple sentences.

The graph defines the scope within which the token can be used. If no graph is specified, the token becomes a global token, applicable to all endpoints.

File: modules/user-access/pages/jwt-token.adoc [hunk 4]
```diff
@@ -492,3 +492,63 @@ gadmin restart -y
...
+    if there is no graph specified the token is global token which is applied to all endpoints
+    
+ * `lifetime`: the `lifetime` means the duration of token can be used and  the time unit is in ms
```
Comment (arun-TG-PM) [inline modules/user-access/pages/jwt-token.adoc]: Same as above: The lifetime specifies the duration for which the token can be used, measured in milliseconds. If no lifetime is specified, the default duration is 7 days.

File: modules/user-access/pages/jwt-token.adoc [hunk 5]
```diff
@@ -492,3 +492,63 @@ gadmin restart -y
...
+
+Drop specific JWT Tokens: add JWT tokens to current user’s block list.
+
+[console]
+----
+DELETE :14240/gsql/v1/tokens"  -d '{"tokens":"$token1, $token2, …"}' 
+----
+
+==== Required Privilege:
+There is no privilege needed since drop JWT token operation is for current user who logins.
```
Comment (arun-TG-PM) [inline modules/user-access/pages/jwt-token.adoc]: For more clarity: No privilege is required to drop JWT tokens, as this operation is limited to the current logged-in user.

**Review by arun-TG-PM — COMMENTED** [review]
Great work! Ensure all GSQL commands, API methods, and section headings are consistently formatted and follow consistent upper/low case headings.

Try to avoid  "you", "we" and "There is/are".

File: modules/user-access/pages/jwt-token.adoc (hunk 5 above)
Comment (zhihuapengtg) [reply modules/user-access/pages/jwt-token.adoc]: @arun-TG-PM  We have addressed your comments, please help to review it again, thanks!

**Review by arun-TG-PM — APPROVED** [review]

**Comment by arun-TG-PM** [comment]
@zhihuapengtg  - Provide me the JIRA ticket

**Comment by zhihuapengtg** [comment]
> @zhihuapengtg - Provide me the JIRA ticket

I think should be this ticket: https://graphsql.atlassian.net/browse/DOC-2227

**Pull request merged**